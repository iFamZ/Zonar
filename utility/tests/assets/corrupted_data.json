{
    "type": "couple of valid data followed by corrupted 'spead' followed by valid data",
    "data": [
        {
            "ts": 1650500050,
            "k": "latitude",
            "v": "47.608013"
        },
        {
            "ts": 1650500050,
            "k": "longitude",
            "v": "-122.335167"
        },
        {
            "ts": 1650500051,
            "k": "spead",
            "v": "60.15"
        },
        {
            "ts": 1650500051,
            "k": "latitude",
            "v": "47.608013"
        },
        {
            "ts": 1650500051,
            "k": "longitude",
            "v": "-122.335160"
        }
    ]
}