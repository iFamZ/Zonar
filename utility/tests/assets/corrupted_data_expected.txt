{'latitude': '47.608013', 'longitude': '-122.335167', 'speed': None, 'ts': 1650500050}
Error detected
